fetch_submissions.py

```diff
@@ -69,7 +69,10 @@ def login(username, password):
       'next': '/'
   }
 
-  CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  response = CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  if response.url == "https://leetcode.com/accounts/login/":
```

Comment:
I think a comment explaining why you used the `url` field of the `Response` object instead of something else like `status_code` or `reason` would be useful.

Reply:
Fair point. The `status_code` and the `reason` returned in both scenarios are 200 and OK respectively. The `url` field was the only one I could see from the Response object that was noticeably different. I'll mention this in a comment.